.ci/pull_fluent_image.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
You should use: 
```suggestion
#!/bin/sh
```
for best portability.

Reply:
Hi Dan,
Yes, sure. 
Thank you.